### PR TITLE
Feature/art view buttons

### PR DESCRIPTION
**Pull request by Jayquintana**
What (if any) features are you implementing?
-  create back to main and show new image buttons on modal. Style other buttons hover effects

What (if anything) did you refactor?
- Nothing

Were there any issues that arose?
- No

Is there anything that you need from your teammate?
- No

Any other comments, questions, or concerns?
- No

### CODE REVIEW

File: src/ArtDetails/ArtDetails.scss [hunk 1]
```diff
@@ -7,9 +9,44 @@
...
   border-radius: 5px;
-  width: 50%;
-  height: 50%;
+  width: 65%;
+  height: 74%;
+  margin-top: 5%;
   font-size: 30px;
+  overflow-y: auto;
+}
+.go-back-button,
```
Comment (StevieAmb) [inline src/ArtDetails/ArtDetails.scss:18]: Wonderful job with these buttons!

File: src/ArtDetails/ArtDetails.scss [hunk 2]
```diff
@@ -7,9 +9,44 @@
...
+  box-shadow: 5px 5px 0px rgba(0, 0, 0, .25);
+  cursor: pointer;
+text-decoration: none;
+}
+
+.go-back-button:hover,
+.generate-new-button:hover {
+  background-color: red;
+  cursor: pointer;
+}
```
Comment (mr-ryan12) [inline src/ArtDetails/ArtDetails.scss:38]: The hover effects on the buttons is a nice touch!

**Review by tonycarpenter21 — APPROVED** [review]
Great job! I like the hover color change on the buttons.

File: src/ArtDetails/ArtDetails.scss [hunk 3]
```diff
@@ -7,9 +9,44 @@
...
+  margin: 25px;
+  border-radius: 5px;
+  text-transform: uppercase;
+  font-weight: 900;
+  box-shadow: 5px 5px 0px rgba(0, 0, 0, .25);
+  cursor: pointer;
+text-decoration: none;
+}
+
+.go-back-button:hover,
```
Comment (StevieAmb) [inline src/ArtDetails/ArtDetails.scss:34]: This is a really beautiful effect.

File: src/ArtDetails/ArtDetails.scss [hunk 4]
```diff
@@ -7,9 +9,44 @@
...
+.generate-new-button:hover {
+  background-color: red;
+  cursor: pointer;
+}
+
+.button-container {
+  // position:absolute;
```
Comment (mr-ryan12) [inline src/ArtDetails/ArtDetails.scss:41]: Probably could remove this at some point.

File: src/ArtDetails/ArtDetails.tsx [hunk 5]
```diff
@@ -25,15 +26,21 @@ interface Details {
...
+                                    : <p className="percent-message">You and {Math.floor(Math.random() * 100) + 1 + '%'} of other people agree! Congratulations!</p>
   return (
     <section className="art-details-container">
-      <h2 className='title-details'>TITLE:</h2><p className="art-details-text">{props.artDetails.title}</p>
-      <h2 className='date-details'>STARTING DATE:</h2><p className="art-details-text">{props.artDetails.date_start}</p>
-      <h2 className='ending-date-details'>ENDING DATE:</h2><p className="art-details-text">{props.artDetails.date_end}</p>
-      <h2 className='artists-details'>ARTIST:</h2><p className="art-details-text">{props.artDetails.artist_title}</p>
-      <h2 className='description-details'>DESCRIPTION:</h2><p className="art-details-text">{props.artDetails.thumbnail.alt_text}</p>
-      {likedMessage}
+      <div className='main-details'>
```
Comment (StevieAmb) [inline src/ArtDetails/ArtDetails.tsx:32]: Great job with adding these buttons! They're awesome!

**Review by StevieAmb — APPROVED** [review]

**Pull request merged**